ci/cron/src/Docs.hs

```diff
@@ -71,7 +72,9 @@ shell_ cmd = do
     Control.void $ shell cmd
 
 proc_ :: [String] -> IO ()
-proc_ args = Control.void $ proc args
+proc_ args = Control.void $ do
+    print args
```

Comment:
are you planning to merge this? Not really opposed but maybe add some more context, even just `"executing command: " <> show args"`